docs/shared/reference/releases.md

```diff
@@ -0,0 +1,39 @@
+# Releases
+
+Major Nx versions are released as the _latest_ every six months, typically around April and October.
+After each major version release, the _previous_ major version moves to long-term support (LTS) for 12 months, after
+which it becomes unsupported.
+Each major version has 18 total months of support from when it is first released to when it falls out of LTS.
+Workspaces should only use major versions that are under LTS.
+
+## Supported Versions
```

Comment:
I think we should include a bit on the difference between Major, Minor, and Patch releases.

Mostly highlighting that most breaking changes are reserved for major versions. Minor versions new features and patch versions include bug fixes. Updating to the latest minor or patch version should be safe to do regularly.